pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_release_jobs.go

```diff
@@ -9,12 +9,20 @@ import (
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
-func GetPayloadTagFromProwJob(prowJob *prowjobv1.ProwJob) string {
-	return prowJob.Labels["release.openshift.io/analysis"]
+// GetPayloadTagFromProwJob gets the payload tag from prow jobs. For aggregated jobs,
+// the label is release.openshift.io/analysis. For non-aggregated jobs, the label
+// is release.openshift.io/tag
+func GetPayloadTagFromProwJob(prowJob *prowjobv1.ProwJob, isAggregated bool) string {
+	payloadTag := prowJob.Labels["release.openshift.io/analysis"]
+	if len(payloadTag) == 0 && !isAggregated {
+		payloadTag = prowJob.Annotations["release.openshift.io/tag"]
```

Comment:
Can we simply always use this value?

Reply:
Good idea. That removes the isAggregated flag as well.

Reply:
Turned out this is still needed. Otherwise we will be picking up the informing run of this job as well.